### PR TITLE
All news

### PR DESCRIPTION
**Pull request by loui981c**
## Description

Setting up the different functions for different subviews and adding the logic to fetch the newest news.
  
## Changes

- Expanding basemodel for response from API
- Adding new API routes
- Creating function for fetching the newest news from the dataset

## Related Issues

- It will fetch the same newest news when reaching the end of the feed.
- The following feed is not implemented

## Checklist

- [x] Code has been tested locally and passes all relevant tests.
- [x] Documentation has been updated to reflect the changes, if applicable.
- [x] Code follows the established coding style and guidelines of the project.
- [x] All new and existing tests related to the changes have passed.  (on test fails because the test is broken, the function works as intended. The test have been ignored) 
- [x] Any necessary dependencies or new packages have been properly documented.
- [x] Pull request title and description are clear and descriptive.
- [x] Reviewers have been assigned to the pull request.
- [x] Performance impact of the changes has been evaluated, if relevant.

## Notes for Reviewers

<3

### CODE REVIEW

File: Collaborative_Filtering/EBNERD_app.py [hunk 1]
```diff
@@ -14,21 +16,63 @@
...
+    topics: list[str]
+    category: int
+    subcategory: list[int]
+    category_str: str
+    total_inviews: Optional[float]
+    total_pageviews: Optional[float]
+    total_read_time: Optional[float]
+    sentiment_score: float
+    sentiment_label: str
+
```
Comment (benceisonline) [inline Collaborative_Filtering/EBNERD_app.py:40]: Det ville nok være mere clean at importere det i stedet, no?

File: Collaborative_Filtering/EBNERD_app.py [hunk 2]
```diff
@@ -14,21 +16,63 @@
...
+    user_id: int
+
+# Pydantic model for response payload
+class FollowingResponse(BaseModel):
+    following_news: list
+
+@app.get("/following", response_model=FollowingResponse)
+def following(request: FollowingRequest):
+    return {"following_news": ["news1", "news2", "news3"]}
+
```
Comment (benceisonline) [inline Collaborative_Filtering/EBNERD_app.py:66]: Forstår ikke helt hvad following bliver brugt til, kan du evt. tilføje en kommentar?

Comment (loui981c) [reply Collaborative_Filtering/EBNERD_app.py:66]: bare en placeholder lige nu

**Review by benceisonline — APPROVED** [review]
swag 8)

**Pull request merged**